### DISCUSSION TITLE
Welcome to picasso Discussions!

### DISCUSSION THREAD

**Discussion by straussmaximilian** [jungmannlab/picasso#211] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andrewtck96** [comment]
Hi @straussmaximilian , 

I am trying to use Picasso to analyse a set of images from a TIRF microscope.

I installed Picasso release v0.3.3 from the releases page (https://github.com/jungmannlab/picasso/releases/). 
When trying to open a RAW video file, it shows the error in the image below:
![image](https://user-images.githubusercontent.com/81626398/142987589-e1cf6afb-11e3-4117-8154-69c5adc39ca5.png)

The RAW video file was saved from a stack of TIFF images from ImageJ, and contains 20000 images. 

What should I do to resolve this issue? 

Thanks and regards,
Andrew

**Reply by straussmaximilian** [reply]
Hi, this looks like a bug. Would you mind uploading and sharing the file so I can test and troubleshoot?

**Reply by andrewtck96** [reply]
Hi @straussmaximilian ,

I can't upload the files directly here, so I've put them on Google Drive. Here's the link: 
https://drive.google.com/drive/folders/1iTnkGuXgedfoUCBDVnPR2IZPtPTMTHK4?usp=sharing

I have uploaded both the original TIFF image stack, and the RAW file that I saved the TIFF stack as from ImageJ. 

Thank you!

Cheers,
Andrew

**Reply by straussmaximilian** [reply]
Hi Andrew,
I checked your file and couldn't reproduce the error. I opened your file in ImageJ and checked the file settings which I then used to import the raw file in localize. This worked.
Do you mind checking if this works for you?
<img width="290" alt="Screenshot 2021-11-25 at 18 07 14" src="https://user-images.githubusercontent.com/16521288/143482609-c2dc3195-f627-4068-bc85-054e3054a0c5.png">

**Reply by andrewtck96** [reply]
Hi @straussmaximilian ,

Thanks for confirming that the file is working fine.

I found out that the issue was caused by the file being stored on a USB thumb drive; it was saved there from the microscope. Probably there was some issue with the read/write permissions of the USB.
I copied the file to my computer desktop, and was then able to open it with Localize.

Thanks again for your help!

Cheers, 
Andrew

**Comment by syho1990** [comment]
Hi @straussmaximilian ,

Does the Picasso support BigTiff format nowadays? I am seeing the following error whenever I open my image with Localize. 
![image](https://user-images.githubusercontent.com/116984240/198856896-a68f42c2-ed05-4101-8aff-7359697b2034.png)
My files are 17 GB with 30,000 frames. When I divide them into 6,000 frame (<4 GB), there is no issue opening the file. 

Thank you,
Sheng-Yang

**Comment by syho1990** [comment]
Dear @straussmaximilian,

I also tried re-slicing them into 10,000 frame (5.5GB) and 29,999 frames (17GB). The Localize app can open those two file without the error message popping out, but will only have the first frame loaded. I attached my original tiff file in the following link: https://ucdavis.box.com/s/mtclwo8bovlxzo99bn3o260okmtl7jpx

Thank you,
Sheng-Yang

**Comment by straussmaximilian** [comment]
Hi,
There has been no support for BigTiff until now. Maybe @rafalkowalewski1 knows if this is planned.
The only workaround is to use the split ome.tif files as MicroManager produces them or to open e.g., in ImageJ and export as raw and use the raw importer.
The frame number itself should not be limiting: with MicroManager files we analyzed datasets with 100k frames +.

**Reply by rafalkowalewski1** [reply]
Hi,

I can try to implement the support for BigTiff in the future versions.
